src/mon/OSDMonitor.cc

```diff
@@ -8491,88 +8490,73 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       return prepare_set_flag(op, CEPH_OSDMAP_NODEEP_SCRUB);
     else if (key == "notieragent")
       return prepare_set_flag(op, CEPH_OSDMAP_NOTIERAGENT);
-    else if (key == "sortbitwise") {
+    else if (key == "sortbitwise" ||
+      key == "recovery_deletes" ||
+      key == "require_jewel_osds" ||
+      key == "require_kraken_osds") {
```

Comment:
It would be much appreciated if these comparisons were aligned, much like we've been doing in the rest of the file when similar comparisons are made.

Reply:
O.k, will do. Thanks!